### DISCUSSION TITLE
How to make "saved searches" available to "collaborators"

### DISCUSSION THREAD

**Discussion by pamelaxu213** [broadinstitute/seqr#3014] [General]
Hi, as I was testing our local install with my team, we noticed that every time a customized search is saved, only the account that created it is able to see it in future variant searches. It'd be a great feature to have if other managers/collaborators on the same project can use the same customized search criteria. Is it possible to enable this function on our end?

**Comment by hanars** [comment]
In the /admin view, edit the VariantSearch model to set the created_by value to null. This will make the search available to all users

**Reply by pamelaxu213** [reply]
Thanks so much, it worked :)

**Reply by anjalijain22** [reply]
Hi Hana! I had a follow up question to this - is it possible to make a saved search available just to manager/collaborators of a particular project and not to all users of seqr?

**Reply by hanars** [reply]
no. searches can be available to either a single user, or to all users

**Reply by anjalijain22** [reply]
Okay, no problem! Thanks for your prompt response!
